packages/rx_bloc_cli/mason_templates/bricks/lib_social_logins/__brick__/lib/lib_social_logins/services/social_login_service.dart

```diff
@@ -8,20 +8,23 @@ abstract class SocialLoginService {
 
   final UserAccountService _userAccountService;
 
-  Future<AuthTokenModel> authenticate();
+  Future<AuthTokenModel?>? authenticate();
 
-  Future<AuthTokenModel> login() async {
+  Future<AuthTokenModel?> login() async {
     final authToken = await authenticate();
+    if (authToken != null) {
```

Comment:
As you can see changing the signature of the authenticate method to return nullable value causes a snowfall of not needed changes. Please try to stick to the non-nullable value